### ISSUE TITLE
"Subscribe to updates" button doesn't have a focus outline

### ISSUE THREAD

**Issue by MMK21Hub** [modrinth/code#1532]
### Describe the bug

On the sign up page, in the first-party auth section, there's a button (styled as a checkbox) that says "Subscribe to updates about Modrinth". However, when it is focused on, there is no visual indication of its focus state.

This issue also affects the welcome page, which has an identical button.

### Steps to reproduce

1. Go to https://modrinth.com/auth/sign-up
2. Click on the "Confirm password" field
3. Press <kbd>Tab</kbd> to move focus to the "subscribe to updates" button

### Expected behavior

There would be a focus ring around the whole button (or maybe just the checkbox icon)

### Additional context

![Screenshot of the main card on the "Welcome to Modrinth!" page](https://github.com/modrinth/knossos/assets/50421330/bb71baa8-d056-469e-b74d-3193a50b43e7)
![Screenshot of the main card on the auth sign-up page](https://github.com/modrinth/knossos/assets/50421330/a8a25423-85db-43b1-812e-9561cbdd7a31)

**Comment by MagnusHJensen** [comment]
This seems fixed:

<img width="456" height="157" alt="Image" src="https://github.com/user-attachments/assets/35279d15-a8d7-451e-b729-eeaf14cf8cc3" />
